WordPress/UITestsFoundation/Screens/DomainsSuggestionsScreen.swift

```diff
@@ -29,4 +29,16 @@ public class DomainsSuggestionsScreen: ScreenObject {
         XCTAssertTrue(DomainsSuggestionsScreen.isLoaded(), "\"Domains suggestions\" screen isn't loaded.")
         return self
     }
+
+    @discardableResult
+    public func selectDomain() throws -> Self {
```

Comment:
Since the screen contains `Select domain` button, it's easy to assume that the function taps this button. However, it does a different thing, which is tapping the last cell. WDYT of renaming the function?

```suggestion
    public func tapLastDomain() throws -> Self {
```
?